conans/test/functional/hooks/hook_test.py

```diff
@@ -234,11 +250,15 @@ def complete_hook_test(self):
         self._check_download_package(conanfile_cache_path, client.out)
 
         client.run("remove * --force")
+        self._check_init_hook(client.out)
         client.run("install basic/0.1@danimtb/testing")
         self._check_download_recipe(conanfile_cache_path, client.out)
         self._check_download_package(conanfile_cache_path, client.out)
         self._check_package_info(client.out)
 
+    def _check_init_hook(self, out):
```

Comment:
My main problem with this test is that it is a bit shallow. The main reason for this hook is to be able to do a ``conan config install`` with a given frequency (daily, for example). It is not clear at all which interface should be used that could call ``conan config install`` from there. Calling ``conan directly`` would cause an infinite recursion. The api is not passed and even if we pass it it is not documented, so we cannot document this feature either.  We could discuss if passing the python api is a possibility, but certainly need to be carefully considered.

Reply:
I'm a little bit confused about why we can't document feature as - runs for each command invocation or something?